accepted/0000-registry-only-dependencies.md

```diff
@@ -0,0 +1,167 @@
+### References
+relates to #581
+
+----
+
+# Registry Only Dependencies
+
+## Summary
+
+When auditing dependencies with `npm audit`, the npm CLI should have a mechanism for communicating (and optionally failing on) dependencies that _do not_ come from a registry, like a [git URL](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#git-urls-as-dependencies).
+
+> _**Note**: this RFC has a hard dependency on [`npm query`](https://github.com/npm/cli/pull/5000) landing to support its implementation._
```

Comment:
Given that `npm query` has already landed I believe this note can be removed from the RFC text

```suggestion
```